chronograf.go

```diff
@@ -191,17 +191,17 @@ type Response interface {
 
 // Source is connection information to a time-series data store.
 type Source struct {
-	ID                 int    `json:"id,string"`                    // ID is the unique ID of the source
-	Name               string `json:"name"`                         // Name is the user-defined name for the source
-	Type               string `json:"type,omitempty"`               // Type specifies which kinds of source (enterprise vs oss)
-	Username           string `json:"username,omitempty"`           // Username is the username to connect to the source
-	Password           string `json:"password,omitempty"`           // Password is in CLEARTEXT
-	SharedSecret       string `json:"sharedSecret,omitempty"`       // ShareSecret is the optional signing secret for Influx JWT authorization
-	URL                string `json:"url"`                          // URL are the connections to the source
-	MetaURL            string `json:"metaUrl,omitempty"`            // MetaURL is the url for the meta node
-	InsecureSkipVerify bool   `json:"insecureSkipVerify,omitempty"` // InsecureSkipVerify as true means any certificate presented by the source is accepted.
-	Default            bool   `json:"default"`                      // Default specifies the default source for the application
-	Telegraf           string `json:"telegraf"`                     // Telegraf is the db telegraf is written to.  By default it is "telegraf"
+	ID                 int      `json:"id,string"`                    // ID is the unique ID of the source
+	Name               string   `json:"name"`                         // Name is the user-defined name for the source
+	Type               string   `json:"type,omitempty"`               // Type specifies which kinds of source (enterprise vs oss)
+	Username           string   `json:"username,omitempty"`           // Username is the username to connect to the source
+	Password           string   `json:"password,omitempty"`           // Password is in CLEARTEXT
+	SharedSecret       string   `json:"sharedSecret,omitempty"`       // ShareSecret is the optional signing secret for Influx JWT authorization
+	URL                string   `json:"url"`                          // URL are the connections to the source
+	MetaURL            []string `json:"metaUrl,omitempty"`            // MetaURL is the urls for the meta nodes
```

Comment:
js people don't like `null`s but would rather this be set to an empty array.  make sure this is done on response.